node/tools/src/k8s.rs

```diff
@@ -16,11 +16,29 @@ use kube::{
     Api, Client, ResourceExt,
 };
 use std::collections::{BTreeMap, HashMap};
+use std::net::SocketAddr;
+use std::str::FromStr;
 use tokio_retry::strategy::FixedInterval;
 use tokio_retry::Retry;
 use tracing::log::info;
+use zksync_consensus_roles::node;
 use zksync_protobuf::serde::Serde;
 
+/// Consensus Node Representation
+#[derive(Debug)]
+pub struct ConsensusNode {
```

Comment:
This should eventually contain the RPC client and all other means of communicating with / using the node programmatically in the test framework.

Reply:
Yes, and I'm also refactoring it in another PR to contain the full configuration and keys to be provided by command line in pod entrypoints, in order to build the image without configuration files.